content/docs/components-and-props.md

```diff
@@ -182,7 +182,7 @@ function Comment(props) {
   return (
     <div className="Comment">
       <div className="UserInfo">
-        <Avatar user={props.author} />
+        <Avatar user={props.user} />
```

Comment:
```suggestion
        <Avatar user={props.author} />
```
This is good. It doesn't require any changes. I would suggest you go through the documentation first. ([here](https://reactjs.org/docs/components-and-props.html#extracting-components)).

For a better understanding of what's actually happening and why we are using `author` and not `user` in `Comment` component, refer ([this](https://codepen.io/pen?&editable=true&editors=0010)) CodePen.